git/foundations_git/commit_messages.md

```diff
@@ -20,14 +20,47 @@ Yes! Let me give you a quick list of reasons why:
 
 -   Having a good commit message history will also be helpful if you come back to a project you were working on after stepping away from it for a while. You likely won't remember your thought process and changes made when initially writing out the code. 
 
+### Bad vs. Good Commits
+
+When it comes to writing commits, it is crucial to know how to write them effectively. Here's an example of a bad commit message:
+
+`fix a bug`
+
+Even though it describes what you did, the message is too vague, which leaves the other developers on your team confused. A good commit message will explain the why behind your changes. In other words, a commit message describes what problem your changes solve and how it solves them. 
```

Comment:
Nit: just to add emphasis to the "why" and make it clear what the most important take away from this text is.

```suggestion
Even though it describes what you did, the message is too vague, which leaves the other developers on your team confused. A good commit message will explain the **why** behind your changes. In other words, a commit message describes what problem your changes solve and how it solves them. 
```